src/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -18,19 +18,24 @@
     <Compile Include="$(CommonPath)\Interop\Windows\advapi32\Interop.RegistryOptions.cs">
       <Link>Common\Interop\Windows\Interop.RegistryOptions.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Kernel32\Interop.RegistryValues.cs">
+      <Link>Common\CoreLib\Interop\Windows\Kernel32\Interop.RegistryValues.cs</Link>
```

Comment:
All registry APIs are defined in Advapi32. The related constants should be defined in Advapi32 as well. I do not think we want to be moving these to Kernel32.

(I am sorry that I have not caught this in the previous PR.)

Reply:
np i will move everything to kernel32

Reply:
Did you meant to say Advapi32 ?